openssl/src/nid.rs

```diff
@@ -104,7 +104,7 @@ pub enum Nid {
     G,
     S,
     I,
-    UID,
+    uid, // uniqueIdentifier
```

Comment:
Well, openssl references them as `uid` and `UID` as included in this PR. However, there are probably issues with many more of the Nid values, and perhaps we should plan to update them all at once for the next breaking release. In the mean time, I can just change the new variant to UserId. What do you think?